cli/Valet/Site.php

```diff
@@ -570,11 +570,33 @@ public function trustCertificate($crtPath, $url)
             $crtPath
         ));
 
-        $this->cli->run(sprintf(
-            'certutil -d $HOME/.mozilla/firefox/*.default -A -t TC -n "%s" -i "%s"',
-            $url,
-            $crtPath
-        ));
+        foreach ($this->getFirefoxProfilePath() as $path) {
+            $this->cli->run(sprintf(
+                'certutil -d ' . $path . ' -A -t TC -n "%s" -i "%s"',
+                $url,
+                $crtPath
+            ));
+        }
+    }
+
+    /**
+     * get profiles paths of firefox
+     *
+     * @return array
+     */
+    private function getFirefoxProfilePath() {
+        $firefoxPath = getenv('HOME').'/.mozilla/firefox/';
+        $profiles = 'profiles.ini';
+        $path = [];
+        if(file_exists($firefoxPath . $profiles)){
+            $ini = parse_ini_file($firefoxPath . $profiles, true);
+            foreach($ini as $key => $value){
+                if(str_starts_with($key, 'Profile')){
```

Comment:
```suggestion
                if(false !== strpos($key, 'Profile')){
```
I suggest not using PHP 8 feature; if the user isn't using that version, it will fail.

Reply:
> I suggest not using PHP 8 feature;

Right.

Maybe in this case is not important, but if you use as is, the condition is a string contains instead string start. So, I think is better use this

```php
if(0 === strpos($key, 'Profile')){
```

Reply:
Yep, my bad! `strpos` works